data/Invert_Matrix_attempt2/Invert_Matrix_attempt2.py

```diff
@@ -0,0 +1,14 @@
+
+
+import numpy as np
+
+x = np.array([[1,-1], [2,6]])
+m_i = invert_matrix(x)
+x_i = np.array([[0.75,0.125], [-0.25,0.125]])
+
+print (x)
+result = True if np.array_equal(m_i,x_i) == True else False
```

Comment:
`np.array_equal` returns True or False. So, you can skip `True if np.array_equal() == True else False`